spec/spec_helper.rb

```diff
@@ -9,6 +9,7 @@
 # in spec/support/ and its subdirectories.
 Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
 
+ActiveRecord::Migration.maintain_test_schema!
```

Comment:
I'd like to have this change in a separate commit, but it seems that we currently need this in order to run tests. Needs a fix at some point, though.